test/TabsSpec.js

```diff
@@ -493,7 +493,7 @@ describe('Tabs', () => {
     );
     assert.equal(ReactDOM.findDOMNode(instance).getAttribute('class'), 'my-tabs-class');
     assert.equal(ReactDOM.findDOMNode(instance).getAttribute('id'), 'my-tabs-id');
-    assert.deepEqual(ReactDOM.findDOMNode(instance).getAttribute('style'), 'opacity:0.5;');
+    assert.deepEqual(ReactDOM.findDOMNode(instance).getAttribute('style'), 'opacity: 0.5;');
```

Comment:
This is already fixed on `master`. I didn't merge this in because the Teaspoon-based checks were failing.

Maybe best to merge in `master`, then update the deps?